.github/workflows/Image.yaml

```diff
@@ -0,0 +1,49 @@
+name: Multi_Arch Images series build
+on: 
+  push:
+    branches:
+      - 'master'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: docker login
+      env:
+        DOCKER_USER: ${{ secrets.DOCKER_USER }}   
+        DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }} 
+        DOCKER_BUILDKIT: 1
+        BUILDX_PLATFORMS: linux/amd64,linux/arm64,linux/ppc64le,linux/s390x,linux/386,linux/arm/v7,linux/arm/v6 
+      run: |
+        docker login -u $DOCKER_USER -p $DOCKER_PASSWORD
+    - name: install buildx
+      id: buildx
+      uses: crazy-max/ghaction-docker-buildx@v1
+      with:
+        version: latest
+
+    - name: building minimal Linux distributions with Linuxkit
+      run: |
+        docker run --rm --privileged linuxkit/binfmt:v0.8
+        docker buildx create --use --name=qemu
+        docker buildx inspect --bootstrap
+
+    - name: docker build builder
+      run: |
```

Comment:
Please use make commands directly. 
You can also set any required variables.

```
REPO=fission TAG=latest make multiarch-bundle
```

```
REPO=fission TAG=latest make image-multiarch
```

Reply:
@sanketsudake should we have tag as latest or release name?

Reply:
If its release, this flow should be triggered manually or tag creation.